main.cpp

```diff
@@ -97,6 +123,10 @@ int		drag_tap_flag=-1;
 int		tm=0,line_tm=-1;
 //int		backup_flag=0;
 BOOL	clipboard_flag=FALSE;
+// new vars
+int		loope_dynamic=0;
+int		select_preset=0;
+BOOL		toggle_pen_col=FALSE;

```

Comment:
```suggestion
BOOL		toggle_pen_color=FALSE;
```

Could you change this variable name (and all references to this variable) as `col` is somewhat misleading, can also be read as "column".

Reply:
done